packages/sdk/src/math/vector2.ts

```diff
@@ -7,10 +7,15 @@ import { Epsilon, Matrix, Scalar, Vector3 } from '.';
 
 // tslint:disable:member-ordering variable-name one-variable-per-declaration trailing-comma no-bitwise curly max-line-length
 
+export interface Vector2Like {
+    x: number;
+    y: number;
+}
+
 /**
  * Class representing a vector containing 2 coordinates
  */
-export class Vector2 {
+export class Vector2 implements Vector2Like {

```

Comment:
>Vector2 [](start = 13, length = 7)

This class also needs a toJSON() method. Copy from Vector3 along with the comment above it.

Reply:
Oh it already had one. Missed that.

---
In reply to: [253243411](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/164#discussion_r253243411) [](ancestors = 253243411)